jdi-light-vuetify-tests/src/main/java/io/github/com/pages/ChipsPage.java

```diff
@@ -49,24 +41,15 @@ public class ChipsPage extends VuetifyPage {
     @UI("#IconChip > .v-chip")
     public static List<Chip> iconChips;
 
-    @UI(".col-12 input[type=text]")
-    public static TextField customListSearchField;
-
     @UI("#CustomListChip [role=listitem]")
     public static WebList customListItems;
 
     @UI("#CustomListChip .v-chip")
     public static List<Chip> customListChipsList;
 
-    @UI("//span[text()[contains(.,'Next')]]")
-    public static Button customListNextButton;
-
     @UI("#ExpandableChip .v-chip")
     public static Chip expandableChip;
 
-    @UI(".v-card .v-list-item__subtitle")
-    public static List<Text> expandableChipEmails;
-
     @UI("div[role='menu']")
     public static UIElement expandableMenu;
```

Comment:
I suppose this element is Menu, so in test we can use Menu's methods

Reply:
Menu is available for all pages, so it should be a part of VuetifyPage

Reply:
It's a menu, which opens when we click on the expandable chip